build.gradle

```diff
@@ -8,14 +9,14 @@ buildscript {
     }
     dependencies {
         classpath("org.springframework.boot:spring-boot-gradle-plugin:1.2.3.RELEASE")
-        classpath 'de.richsource.gradle.plugins:gwt-gradle-plugin:0.6'
+        classpath 'de.richsource.gradle.plugins:gwt-gradle-plugin:0.7-SNAPSHOT'
```

Comment:
Version `0.6` does not support the GWT 2.8 `generateJsInteropExports` compiler flag.